services/src/main/java/io/trello/trelloapp/card/CardController.java

```diff
@@ -19,4 +21,9 @@
 
         return new ResponseEntity<CardListModel>(cardLisResponse, HttpStatus.OK);
     }
+
+    @PostMapping(value="/api/new-card")
+    public ResponseEntity<CardModel> addNewCard(@RequestBody CardModel newCardInfo) {
+        return new ResponseEntity<CardModel>(newCardInfo, HttpStatus.OK);
```

Comment:
Will making the `ArrayList<CardModel> Cards`, as class property and appending the request object into the array, make the get request at give the updated list ? If that is possible could you do that ?

Reply:
plan for h2 db (in-memory)